Sources/PointFree/RouteNotFoundMiddleware.swift

```diff
@@ -11,21 +11,25 @@ import Styleguide
 import Tuple
 import View
 
-let routeNotFoundMiddleware =
-  currentUserMiddleware
-    >=> currentSubscriptionMiddleware
-    >=> writeStatus(.notFound)
-    >=> map(lower)
-    >>> respond(
-      view: routeNotFoundView,
-      layoutData: { subscription, currentUser in
-        SimplePageLayoutData(
-          currentUser: currentUser,
-          data: unit,
-          title: "Page not found"
-        )
-    }
-)
+func routeNotFoundMiddleware<A>(
+  _ conn: Conn<StatusLineOpen, A>
```

Comment:
I need to generalize this middleware over `A` so that I could plug it into places that don't only work with `Unit`.